mappings/net/minecraft/client/render/world/WorldRenderer.mapping

```diff
@@ -71,11 +97,31 @@ CLASS net/minecraft/class_2346 net/minecraft/client/render/world/WorldRenderer
 		ARG 1 camera
 		ARG 2 view
 		ARG 3 tickDelta
-	METHOD method_9529 (Lnet/minecraft/class_1752;Z)Z
+	METHOD method_9529 prepareChunks (Lnet/minecraft/class_1752;Z)Z
 		ARG 1 camera
+		ARG 2 dontDoExtraLogic
 	METHOD method_9530 renderBlockOutline (Lnet/minecraft/class_2674;Lnet/minecraft/class_2253;IF)V
+		ARG 1 player
+		ARG 2 tergetedBlock
+		ARG 3 alwaysZero
```

Comment:
Similar to the comment on `alwaysFalse`, if code is unused or has no clear purpose, it's best left unmapped.

Reply:
typo `tergeted` -> `targeted`